elasticdl/pkg/kernel/capi/kernel_api.h

```diff
@@ -0,0 +1,5 @@
+#ifndef ELASTICDL_KERNEL_API_H_
+#define ELASTICDL_KERNEL_API_H_
+
+void SGD(float* grad, float* param, double lr, long long size);
+#endif
```

Comment:
The Google C++ style guide has some requirements on guard https://google.github.io/styleguide/cppguide.html#The__define_Guard

Reply:
Done